types/bayes-classifier/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for bayes-classifier 0.0
+// Project: https://github.com/miguelmota/bayes-classifier
+// Definitions by: Jason Harrison <https://github.com/jasonharrison>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+export interface Classifications {
+    label: string;
+    value: number;
+}
+
+export default class bayes_classifier {
```

Comment:
The BayesClassifier type in the source project is exported with `module.exports = BayesClassifier`, so you need to use the equivalent TS syntax: `export = BayesClassifier`.

Reply:
I made this change but had to edit `tsconfig.json` and enable `esModuleInterop = true` to make things pass.  Can you check things over to make sure my changes were correct?  Thanks!

Reply:
Sorry, I forgot to mention that the test needs to use a commonjs import to match the commonjs export when `esModuleInterop: "false"`: `import BayesClassifier = require('bayes-classifier')`.

Note that *most* users will have `esModuleInterop: "true"` these days, and thus be able to use `import BayesClassifier from 'bayes-classifier'`, but for backward compatibility, DT tests should have `esModuleInterop: "false"` and use the old syntax in order to make sure it works for old code bases.

Reply:
Thank you very much for the guidance, @sandersn.  I am relatively new to TS/JS and this is my first type definition contribution.  Please check now.